example.scss

```diff
@@ -5,4 +5,5 @@
 
 .this-was-edited-from-github-flow {
   color: red;
+  border: 1px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.